app/components/content/NuxiMoodGallery.vue

```
@@ -0,0 +1,52 @@
+  { mood: 'excited', label: 'excited' },
+  { mood: 'thinking', label: 'thinking' },
+  { mood: 'surprised', label: 'surprised' },
+  { mood: 'sleeping', label: 'sleeping' }
+]
+</script>
+
+<template>
+  <div class="nuxi-mood-gallery not-prose my-8">
+    <div v-for="item in moods" :key="item.mood" class="nuxi-mood-card">
+      <AgentNuxiIcon class="size-10 text-primary" :mood="item.mood" :interactive="false" />
+      <div class="nuxi-mood-label">{{ item.label }}</div>
```

ESLint

[error] Expected 1 line break after opening tag (`<div>`), but no line breaks found.

(vue/singleline-html-element-content-newline)